rest/src/main/kotlin/builder/channel/ForumChannelCreateBuilder.kt

```diff
@@ -45,6 +49,18 @@ public class ForumChannelCreateBuilder(public var name: String) :
 
     override var permissionOverwrites: MutableSet<Overwrite> = mutableSetOf()
 
+    private var _defaultReactionEmoji: Optional<DiscordDefaultReaction> = Optional.Missing()
+    public var defaultReactionEmoji: DiscordDefaultReaction? by ::_defaultReactionEmoji.delegate()
+
+    private var _defaultThreadRateLimitPerUser: Optional<DurationInSeconds> = Optional.Missing()
+    public var defaultThreadRateLimitPerUser: DurationInSeconds? by ::_defaultThreadRateLimitPerUser.delegate()
```

Comment:
this isn't documented [here](https://discord.com/developers/docs/resources/guild#create-guild-channel-json-params), are you sure it can be used when creating (forum) channels?

Reply:
Yes! I tested, see https://github.com/discord/discord-api-docs/pull/5606